codegen/core/src/main/scala/com/lightbend/akkasls/codegen/Packaging.scala

```diff
@@ -28,10 +28,7 @@ case class FullyQualifiedName(protoName: String, name: String, parent: PackageNa
   lazy val fullQualifiedName = s"${parent.javaPackage}.$name"
   lazy val fullName = {
     if (parent.javaMultipleFiles) name
-    else
-      parent.javaOuterClassnameOption
-        .map(outer => s"$outer.$name")
-        .getOrElse(name)
+    else s"${parent.javaOuterClassname}.$name"
```

Comment:
First part of the fix, types will always be wrapped in the outer class if it's not multiple files. And use this `fullName` method in the generation to support the variations (rather than always adding the outer class).